.changeset/forty-students-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": minor
+---
+
+Pratt parser should no longer panic because it only resolves on valid sequences.
```

Comment:
Not sure if users would be familiar with what "Pratt parser" or its "valid sequences" are, as it is an internal implementation detail. I suggest phrasing it in terms of the public API/features. For example: "parsing operators with missing operands should no longer panic". The changelog automatically includes a link to this PR (where it can/may contain any necessary technical details) if someone wanted to dig deeper.